settings.gradle

```diff
@@ -13,6 +13,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+plugins {
+    id 'com.gradle.enterprise' version '3.13.1'
+    id 'com.gradle.common-custom-user-data-gradle-plugin' version '1.10'
+}
+
+def isGithubActions = System.getenv('GITHUB_ACTIONS') != null
+def isJenkins = System.getenv('JENKINS_URL') != null
+def isCI = isGithubActions || isJenkins
+
+gradleEnterprise {
+    server = "https://ge.apache.org"
+    buildScan {
+        capture { taskInputFiles = true }
+        uploadInBackground = !isCI
+        publishAlways()
+        publishIfAuthenticated()
+        obfuscation {
+            ipAddresses { addresses -> addresses.collect { address -> "0.0.0.0"} }
```

Comment:
Please add a comment here so that future maintainers can understand the motivation of this line.

Something like, "We are obfuscating ip address of the CI hosts here. HostNames will still be available in the scans to detect any Host specific slowness / flaky errors"

Reply:
✅